### ISSUE TITLE
Incorporating Orientations into End-to-end Driving Model for Steering Control. (arXiv:2103.05846v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#4534]
Incorporating Orientations into End-to-end Driving Model for Steering Control. (arXiv:2103.05846v1 [cs.RO])<br>
https://ift.tt/3l4mpey<br>
<p>In this paper, we present a novel end-to-end deep neural network model for autonomous driving that takes monocular image sequence as input, and directly generates the steering control angle. Firstly, we model the end-to-end driving problem as a local path planning process. Inspired by the environmental representation in the classical planning algorithms(i.e. the beam curvature method), pixel-wise orientations are fed into the network to learn direction-aware features. Next, to handle the imbalanced distribution of steering values in training datasets, we propose an improvement on a cost-sensitive loss function named SteeringLoss2. Besides, we also present a new end-to-end driving dataset, which provides corresponding LiDAR and image sequences, as well as standard driving behaviors. Our dataset includes multiple driving scenarios, such as urban, country, and off-road. Numerous experiments are conducted on both public available LiVi-Set and our own dataset, and the results show that the model using our proposed methods can predict steering angle accurately.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
